my_enumerable.rb

```diff
@@ -0,0 +1,17 @@
+module MyEnumerable
+  def all?()
+    each { |element| return false unless yield(element) }
+    true
+  end
+
+  def any?()
+    each { |element| return true if yield(element) }
+    false
+  end
+
+  def filter()
+    result = []
+    each { |element| result << element if yield(element) }
+    result
+  end
```

Comment:
- [ ] Please make sure the result of the filter should return in array containing the result items. Currently, it's returning as individuals items.

### Your result

![image](https://github.com/kalbek/morse-code-decoder/assets/83803414/515755c0-6cbb-4bd4-88da-cc47cd42ecea)

### Expected results 

![image](https://github.com/kalbek/morse-code-decoder/assets/83803414/8c95c36b-696c-436e-94b3-685587b754f5)